src/components/ec-submenu/ec-submenu.vue

```diff
@@ -65,92 +65,85 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/index';
-@import '../../scss/tools/index';
-
-$ec-submenu-body: $level-3-body-and-headings !default;
-$ec-submenu-hover: $level-4-tech-blue !default;
-$ec-submenu-disabled: $level-6-disabled-lines !default;
+<style>
 
 .ec-submenu {
-  position: relative;
+  @apply tw-relative;
 
   &__header {
     min-width: 100px;
-    margin-bottom: 12px;
-    white-space: nowrap;
-    display: flex;
-    flex-direction: column;
-
-    @include media__from-1024 {
-      flex-direction: row;
-      border-bottom: 1px solid $ec-submenu-disabled;
+
+    @apply tw-mb-12;
+    @apply tw-flex tw-whitespace-no-wrap;
+    @apply tw-flex-col;
+
+    @media (min-width: 1024px) {
+      @apply tw-flex-row;
+      @apply tw-border-b tw-border-solid tw-text-gray-6;
     }
   }
 
   &__header-item {
-    bottom: -1px;
-    cursor: pointer;
-    display: inline-block;
-    position: relative;
-    text-align: center;
+    @apply tw--bottom-1;
```

Comment:
typo

Reply:
With the typo you mean the doble '-'?
This is correct because we want a negative pixel

Reply:
oooh, sorry, my fault!

Reply:
Yeah, negative classes are weird in TW. because the negative sign is applied between the prefix and name of the utility. This is happening everywhere, e.g. `tw-m--8` <- `margin: -8px`, meanwhile `tw-m-8` is `margin: 8px`. I'm not sure why tailwind guys decided to do this, I'd rather prefer `-tw-m-8`.

BTW, you cannot have typo in `@apply`, the build will fail if the class doesn't exist.

Reply:
fyi, before posting I checked in https://chameleon.ebury.now.sh/?path=/story/css-tailwind--all but couldn't find the class

Reply:
Yeah, we added the negative numbers in this story: 
```
    inset: (theme, { negative }) => ({
      ...theme('spacing'),
      ...negative(theme('spacing')),
      auto: 'auto',
    }),
```
If you check deployment of this branch, you will see them:
https://chameleon-git-onl-4525-submenu-postcss.ebury.now.sh/?path=/story/css-tailwind--all